RemoteInput/Plugin/Python/PythonPlugin.cxx

```diff
@@ -65,7 +151,7 @@ bool RegisterEnum(PyObject* module, const std::string &enum_name, const std::uno
     return true;
 }
 
-PyMODINIT_FUNC MODINIT(remote_input)()
+PyMODINIT_FUNC PyInit_remote_input()
```

Comment:
Hmm, why did we change this?

Reply:
The macro was only used for py2 vs py3, and since the Python.h changes remove py2 support I figured we could remove them elsewhere.